### ISSUE TITLE
em-calc() is no longer included. Is it on purpose?

### ISSUE THREAD

**Issue by JoelESvensson** [zurb/foundation-sites#7158]
Like the title said

**Comment by NicolasGraph** [comment]
`em-calc()` changed to `rem-calc()` with F5 I think and it works fine, it's even used in *_settings*.

**Comment by JoelESvensson** [comment]
I'm aware of that. em-calc is still usable for those cases when you need to use em units

**Comment by NicolasGraph** [comment]
Ok, I didn't notice that…

**Comment by colin-marshall** [comment]
I don't have an answer as to why it was excluded, but here is the function from F5 if you want to add it to an F6 project:

```scss
@function em-calc($values, $base-value: $rem-base) {
  $remValues: rem-calc($values, $base-value: $rem-base);

  $max: length($remValues);

  @if $max == 1 { @return strip-unit(nth($remValues, 1)) * 1em; }

  $emValues: ();
  @for $i from 1 through $max {
    $emValues: append($emValues, strip-unit(nth($remValues, $i)) * 1em);
  }
  @return $emValues;
}
```

**Comment by JoelESvensson** [comment]
Thanks! I actually just found out that the functionality still exists internally with the function `-zf-bp-to-em()`

**Comment by gakimball** [comment]
`em-calc()` was sort of a late addition to Foundation 5 as I recall, but we scrapped it as a public function in 6. The benefit of `rem-calc()` is that you put a pixel value in and get an exact rem value out, and it can never change. An `em-calc()` doesn't get you that same benefit.

Since we only use em calculation to create breakpoints, we renamed the function, as @JoelESvensson noticed. It's considered a private framework function, but that's not really a concept in Sass, so you're free to use it.
